packages/core/test/unit/lib/markdown-it/highlight/HighlightRuleComponent.test.ts

```diff
@@ -0,0 +1,151 @@
+import { HighlightRuleComponent } from '../../../../../src/lib/markdown-it/highlight/HighlightRuleComponent';
+
+describe('parseRuleComponent', () => {
+  // Test for valid line-part syntax
```

Comment:
Could you remove the comment above each test? Whatever comment should already be incorporated directly into that test description, if not please do so.

Reply:
done